server/index.ts

```diff
@@ -28,6 +29,8 @@ headerSetup(app);
 app.use("/signup", signupRouter);
 app.use("/login", loginRouter);
 app.use("/", router);
+app.use('/api', router);
+app.use("/app/parents", parentsRouter)
```

Comment:
please use "/api/parents" route

Reply:
Ok, then change the file 'routes/app' to 'routes/api' also?